### PR TITLE
Add sample for hierarchical grid excel exporter #2602

### LINKED ISSUE #2602: Create a sample for hierarchical grid export to Excel
Add a new sample demonstrating the export feature.

### PR DESCRIPTION
**Pull request by zhosti**
closes #2602

### CODE REVIEW

**Review by onlyexeption — APPROVED** [review]

File: src/app/data/artistData.ts [hunk 1]
```diff
@@ -0,0 +1,274 @@
...
+                    {
+                        Number: 2,
+                        Title: 'Forever',
+                        Released: new Date('23 Dec 2019'),
+                        Genre: 'Punk',
+                        Album: 'Pushing up daisies'
+                    },
+                    {
+                        Number: 3,
+                        Title: 'Wfdsafsaings Forever - Remix',
```
Comment (onlyexeption) [inline src/app/data/artistData.ts:115]: Change this

File: src/app/data/artistData.ts [hunk 2]
```diff
@@ -0,0 +1,274 @@
...
+                LaunchDate: new Date('May 31, 2020'),
+                BillboardReview: 86,
+                USBillboard200: 42,
+                Artist: 'Naomí Yepes',
+                Songs: [
+                    {
+                        Number: 1,
+                        Title: 'Wood Shavings Forever',
+                        Released: new Date('9 Jun 2019'),
+                        Genre: '*fdasfsa',
```
Comment (onlyexeption) [inline src/app/data/artistData.ts:103]: Change this

File: src/app/data/artistData.ts [hunk 3]
```diff
@@ -0,0 +1,274 @@
...
+        ]
+    },
+    {
+        Artist: 'Chloe',
+        Debut: 2015,
+        GrammyNominations: 3,
+        GrammyAwards: 1,
+        HasGrammyAward: true,
+    }
+];
```
Comment (onlyexeption) [inline src/app/data/artistData.ts:274]: add new line at the end

File: src/app/hierarchical-grid/hierarchical-grid-excel-exporter/hierarchical-grid-excel-export.sample.html [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+            <igx-column field="Location"></igx-column>
+            <igx-column field="Headliner"></igx-column>
+            <igx-row-island [key]="'TourData'" [autoGenerate]="false">
+                <igx-column field="Country"></igx-column>
+                <igx-column field="TicketsSold" header="Tickets Sold"></igx-column>
+                <igx-column field="Attendants"></igx-column>
+            </igx-row-island>
+        </igx-row-island>
+    </igx-hierarchical-grid>
+</div>
```
Comment (onlyexeption) [inline src/app/hierarchical-grid/hierarchical-grid-excel-exporter/hierarchical-grid-excel-export.sample.html:66]: add new line

**Review by onlyexeption — CHANGES_REQUESTED** [review]

**Review by onlyexeption — DISMISSED** [review]

**Comment by gedinakova** [comment]
@zhosti @onlyexeption It seems the topic describes an approach with a button, instead of using the toolbar UI for exporting. We should align the current sample with the topic and make it as similar as possible to the other grids' ones.

**Review by gedinakova — CHANGES_REQUESTED** [review]
We should update the sample to match the other two grids' ones.

**Comment by gedinakova** [comment]
@zhosti There are errors in live-editing apps:
ERROR
Error: StaticInjectorError(AppModule)[HGridExcelExportSampleComponent -> IgxExcelExporterService]:
StaticInjectorError(Platform: core)[HGridExcelExportSampleComponent -> IgxExcelExporterService]:
NullInjectorError: No provider for IgxExcelExporterService!

**Pull request merged**